packages/bls12-381-unix/bls12-381-unix.1.1.0/opam

```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+synopsis: """\
+UNIX version of BLS12-381 primitives implementing the virtual package
+bls12-381 with blst backend"""
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage: "https://gitlab.com/dannywillems/ocaml-bls12-381"
+bug-reports: "https://gitlab.com/dannywillems/ocaml-bls12-381/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.8.4"}
+  "dune-configurator" {build}
+  "zarith" {>= "1.10" & < "2.0"}
+  "bls12-381" {= version}
+  "hex"
+  "alcotest" {with-test}
+  "integers"
+  "bisect_ppx" {with-test & >= "2.5"}
+  "ff-pbt" {>= "0.6.0" & < "0.7.0" & with-test}
+]
+available: arch != "s390x" & arch != "ppc64"
```

Comment:
```suggestion
available: arch != "s390x" & arch != "ppc64]
x-ci-accept-failures: [
  "centos-7" # because of -Werror
  "oraclelinux-7" # because of -Werror
]
```

Reply:
/cc @kit-ty-kate and @dannywillems

Reply:
Why did you remove the closing `"`?
I add `x-ci-accept-failures` upstream.